### PR TITLE
#162197071 Super admin can edit user role

### PR DESCRIPTION
**Pull request by ezrqnkemboi**
#### What does this PR do?

A super admin should be able to edit user roles

#### Description of Task to be completed?
- A super admin should be able to edit user roles
#### How should this be manually tested?

- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change directory - `cd travel_tool_back`
- Checkout to current branch - `git checkout ft-super-admin-edit-user-roles-162197071`
- Run application - `make start`
- Assign yourself the role of super admin in the database
- Use patch request and [this json data](https://gist.github.com/ezrqnkemboi/7306cef85f64cebc6d8a3d0e5e936fba)
- The endpoint for this is `http://127.0.0.1:5000/api/v1/user/role/:id`

#### Any background context you want to provide?

- You must be a super admin to perform this operation
- Make use of patch for this request

#### What are the relevant pivotal tracker stories?

[#162197071](https://www.pivotaltracker.com/story/show/162197071)

#### Screenshots (if appropriate)
screen shot 2018-11-30 at 06 35 47

#### Questions:

### CODE REVIEW

**Comment by asamisellah** [comment]
Kindly fix code climate issues and work to increase the test coverage

**Comment by janetnim** [comment]
screen shot 2018-11-30 at 17 50 25

**Comment by janetnim** [comment]
I've been able to test your work and it works well. However, I noted that I could also be able to edit roles with ids that do not exist. Kindly check on this edge case.

**Comment by ezrqnkemboi** [comment]
Working on that @janetnim .

File: src/modules/userRole/UpdateRoleController.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import models from '../../database/models';
+import CustomError from '../../helpers/Error';
+import UserRoleController from './UserRoleController';
+
+class UpdateUserRoleController {
+  static async updateRole(req, res) {
+    const { params: { id }, body: { roleName, description } } = req;
+    const getRoleId = await models.Role.findOne({
```
Comment (amarachukwu-agbo) [inline src/modules/userRole/UpdateRoleController.js]: Why is line 8 - 17 not within the try block? What happens if an error occurs somewhere in the execution of those lines of code?

**Comment by amarachukwu-agbo** [comment]
@ezrqnkemboi, please fix your commit header

File: src/modules/userRole/UpdateRoleController.js (hunk 1 above)
Comment (ezrqnkemboi) [reply src/modules/userRole/UpdateRoleController.js]: I got the point and a sense of try and catch now. Thank you for letting me know that. I think it is important for this case.
